portal/Cargo.toml

```diff
@@ -14,7 +14,7 @@ version.workspace = true
 [dependencies]
 ashpd = {workspace = true, features = ["backend", "tracing"]}
 clap.workspace = true
-oo7.workspace = true
+oo7 = { workspace = true, default-features = true }
```

Comment:
shouldn't this enable the tokio feature? have you tested that the portal & cli works with your changes?

Reply:
tokio feature is [covered](https://github.com/bilelmoussaoui/oo7/blob/cbc3fa9b8df33634595c9d846dc70fc9b00338ed/client/Cargo.toml#L59) with the `default-features = true` right? I have just tested the cli and it works. I'm not sure how to test the portal.